### PR TITLE
Implementing feedback analytics call

### PR DESCRIPTION
**Pull request by hannahbergam**
Adds an amplitude call when a teacher submits feedback on a level. 

Here is a pic of the console log when called:
Screen Shot 2023-02-07 at 10 45 56 AM

I tested this on this level (to illustrate!): http://localhost-studio.code.org:3000/s/csd1-2022/lessons/7/levels/2?section_id=40&user_id=92
Screen Shot 2023-02-06 at 2 59 03 PM

## Links

- spec: []()
- jira ticket: https://codedotorg.atlassian.net/browse/TEACH-193

## Testing story

Added a test to ensure that the amplitude call happens when submit is pressed.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/templates/instructions/teacherFeedback/EditableTeacherFeedback.jsx [hunk 1]
```diff
@@ -155,6 +157,10 @@ export class EditableTeacherFeedback extends Component {
           submitting: false
         });
       });
+    analyticsReporter.sendEvent(EVENTS.FEEDBACK_SUBMITTED, {
+      sectionId: this.props.selectedSectionId,
+      unitId: this.props.serverScriptId
```
Comment (bethanyaconnor) [inline apps/src/templates/instructions/teacherFeedback/EditableTeacherFeedback.jsx]: Do we also want to send `levelId` here?

File: apps/src/lib/util/AnalyticsConstants.js [hunk 2]
```diff
@@ -28,7 +28,8 @@ const EVENTS = {
   PROGRESS_VIEWED: 'Section Progress Viewed',
   PROGRESS_TOGGLE: 'Section Progress Toggled',
   PROGRESS_CHANGE_UNIT: 'Section Progress Unit Changed',
-  PROGRESS_JUMP_TO_LESSON: 'Section Progress Jump to Lesson'
+  PROGRESS_JUMP_TO_LESSON: 'Section Progress Jump to Lesson',
+  FEEDBACK_SUBMITTED: 'Level Feedback Submitted'
```
Comment (bethanyaconnor) [inline apps/src/lib/util/AnalyticsConstants.js:35]: nit: do we think this should be in its own "section"? like "levels" or something? I can go either way and that's definitely not blocking this PR

**Review by bethanyaconnor — APPROVED** [review]
A couple of small questions but otherwise looks good!

File: apps/src/lib/util/AnalyticsConstants.js (hunk 2 above)
Comment (hannahbergam) [reply apps/src/lib/util/AnalyticsConstants.js:35]: good call- added!

File: apps/src/templates/instructions/teacherFeedback/EditableTeacherFeedback.jsx (hunk 1 above)
Comment (hannahbergam) [reply apps/src/templates/instructions/teacherFeedback/EditableTeacherFeedback.jsx]: haha yeah sure! Added

**Pull request merged**